modules/util/truncate.go

```diff
@@ -6,20 +6,23 @@ package util
 
 import "unicode/utf8"
 
+// in UTF8 "…" is 3 bytes so doesn't really gain us anything...
+const utf8Ellipsis = "…"
+const asciiEllipsis = "..."
```

Comment:
I don't think these constants help clarify the code or help us to understand what's going on better.

However if you really think it helps then fine.

Reply:
"…" and "..." are used in many places (including in unit test), so I introduce these two `const` to help, otherwise, we would copy&paste these two strings many times.